github/repos_forks.go

```diff
@@ -77,6 +79,10 @@ func (s *RepositoriesService) CreateFork(ctx context.Context, owner, repo string
 
 	fork := new(Repository)
 	resp, err := s.client.Do(ctx, req, fork)
+	if _, ok := err.(*AcceptedError); ok {
+		json.NewDecoder(resp.Body).Decode(fork)
+		return fork, resp, err
+	}
 	if err != nil {
```

Comment:
I'd suggest merging this with the if statement above, so it's a little easier to see that this if statement deals with the same `err` value from line 81:

```Go
if _, ok := err.(*AcceptedError); ok {
	...
} else if err != nil {
	...
}
```

Reply:
I think it's fine keeping them separate. They are very different code paths; the `AcceptedError` case is only nominally an error.